cmd/migrate/README.md

```diff
@@ -0,0 +1,57 @@
+# Loki Migrate Tool
+
+**WARNING: THIS TOOL IS NOT WELL TESTED, ALWAYS MAKE BACKUPS AND TEST ON LESS IMPORTANT DATA FIRST!**
+
+This is sort of a bare minimum code hooked directly into the store interfaces within Loki.a
+
+Two stores are created, a source store and dest store.
+
+Chunks are queried from the source store and written to the dest store, new index entries are created in the dest store as well.
+
+This _should_ handle schema changes and different schemas on both the source and dest store, I say should because for the original use this was created this use case was not tested, but there is no reason it shouldn't work for this.
+
+You should be able to:
+
+* Migrate between clusters
+* Change tenant ID during migration
+* Migrate data between schemas
+
+All data is read and re-written (even when migrating within the same cluster), there are really no optimizations in this code for performance, there are much faster ways to move data depending on what you want to change.
+
+This is simple and because it uses the storage interfaces should be complete and should stay working, but it's not optimized to be fast.
```

Comment:
```suggestion
This is simple and because it uses the storage interfaces, should be complete and should stay working, but it's not optimized to be fast.
```